### PR TITLE
Feature/submission string changes

### PR DESCRIPTION
**Pull request by Oliver-Zimmerman**
-Submission string changes. Strings marked final are finalized.
-Added required title to submission done fragment.
-Some strings/screens still need to be finalized

## Checklist

* [X ] Test your changes as thoroughly as possible before you commit them. Preferably, automate your test by unit/integration tests.
* [ ] If this PR comes from a fork, please [Allow edits from maintainers](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork)
* [ ] Set a speaking title. Format: {task_name} (closes #{issue_number}). For example: Use logger (closes #41)
* [ ] [Link your Pull Request to an issue](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) (if applicable)
* [ ] Create Work In Progress [WIP] pull requests only if you need clarification or an explicit review before you can continue your work item.
* [X ] Make sure that your PR is not introducing _unncessary_ reformatting (e.g., introduced by on-save hooks in your IDE)
* [X ] Check our [Contribution Guidelines](https://github.com/corona-warn-app/cwa-app-android/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: Corona-Warn-App/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -341,24 +341,24 @@
     ###################################### -->
     <!-- Permission Rationale Dialog -->
     <string name="submission_qr_code_scan_permission_rationale_dialog_headline">Kamera Berechtigung benötigt</string>
-    <string name="submission_qr_code_scan_permission_rationale_dialog_body">@string/lorem_ipsum</string>
+    <string name="submission_qr_code_scan_permission_rationale_dialog_body">Bitte erlauben Sie die autorisierung der kamera für das scannen von QR-Codes.</string>
```
Comment (m1ga) [inline Corona-Warn-App/src/main/res/values/strings.xml:344]: Bitte erlauben Sie die Autorisierung der Kamera für das Scannen von QR-Codes.

File: Corona-Warn-App/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -341,24 +341,24 @@
...
 
     <string name="submission_qr_code_scan_permission_rationale_dialog_button_positive">Erlauben</string>
     <string name="submission_qr_code_scan_permission_rationale_dialog_button_negative">Nicht erlauben</string>
     <!-- Permission Denied Dialog -->
     <string name="submission_qr_code_scan_permission_denied_dialog_headline">Keine Kamera Berechtigung</string>
-    <string name="submission_qr_code_scan_permission_denied_dialog_body">@string/lorem_ipsum</string>
+    <string name="submission_qr_code_scan_permission_denied_dialog_body">Bitte erlauben Sie die autorisierung der kamera für das scannen von QR-Codes.</string>
```
Comment (m1ga) [inline Corona-Warn-App/src/main/res/values/strings.xml:350]: Bitte erlauben Sie die Autorisierung der Kamera für das Scannen von QR-Codes.

File: Corona-Warn-App/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -368,80 +368,82 @@
...
 
-    <string name="submission_test_result_positive_steps_positive_heading">Testergebnis Positiv</string>
-    <string name="submission_test_result_positive_steps_positive_body">Ihr Testergebnis wurde erfolgreich als “Positiv” verifiziert.</string>
-    <string name="submission_test_result_positive_steps_warning_others_heading">Andere warnen</string>
-    <string name="submission_test_result_positive_steps_warning_others_body">Teilen Sie Ihre Zufallskennungen, damit andere gewarnt werden können.</string>
-    <string name="submission_test_result_positive_continue_button">Weiter</string>
+    <string name="submission_test_result_positive_steps_positive_heading">Ihr Test-Ergebnis</string> <!-- final -->
+    <string name="submission_test_result_positive_steps_positive_body">Ihr Test-Ergebnis wurde erfolgreich als “Positiv” verifiziert.</string> <!-- final -->
+    <string name="submission_test_result_positive_steps_warning_others_heading">Andere warnen</string> <!-- final -->
+    <string name="submission_test_result_positive_steps_warning_others_body">Ihre Zufallscodes der letzten 14 Tageteilen,umanderezu schützen und dieInfektionskette zu unterbrechen.</string> <!-- final -->
```
Comment (m1ga) [inline Corona-Warn-App/src/main/res/values/strings.xml:390]: Ihre Zufallscodes der letzten 14 Tage teilen, um andere zu schützen und die Infektionskette zu unterbrechen.

File: Corona-Warn-App/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -368,80 +368,82 @@
...
+    <string name="submission_positive_dialog_confirmation_negative">Nicht erlauben</string> <!-- final -->
+
     <!-- Submission Done -->
-    <string name="submission_done_headline">Vielen dank!</string>
-    <string name="submission_done_body">Sie haben einen wichtigen Beitrag geleistet! Dank Ihrer Mithilfe können Andere gewarnt werden und entsprechend reagieren.</string>
-    <string name="submission_done_subtitle">Bitte beachten Sie:</string>
-    <string name="submission_done_contagious">Sie sind ansteckend.</string>
-    <string name="submission_done_isolate">Bitte isolieren Sie sich von anderen Personen.</string>
-    <string name="submission_done_button_done">Fertig</string>
+    <string name="submission_done_title">Vielen dank!</string> <!-- final -->
```
Comment (m1ga) [inline Corona-Warn-App/src/main/res/values/strings.xml:434]: Vielen Dank!

File: Corona-Warn-App/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -368,80 +368,82 @@
...
+    <string name="submission_done_headline">Sie haben einen wichtigen Beitrag geleistet!</string> <!-- final -->
+    <string name="submission_done_body">Dank Ihrer Mithilfe können Andere gewarnt werden und entsprechend reagieren.</string> <!-- final -->
```
Comment (m1ga) [inline Corona-Warn-App/src/main/res/values/strings.xml:436]: Dank Ihrer Mithilfe können andere gewarnt werden und entsprechend reagieren.

File: Corona-Warn-App/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -450,7 +452,7 @@
     <string name="test_api_button_share_my_keys">Share my keys via Email</string>
     <string name="test_api_button_enter_other_keys">Submit Exposure Key</string>
     <string name="test_api_button_show_qr_code">Show QR Code</string>
-    <string name="test_api_button_submit_keys">Submit keys to Server</string>
+    <string name="test_api_button_submit_keys">Submit keys to Servier</string>
```
Comment (m1ga) [inline Corona-Warn-App/src/main/res/values/strings.xml:455]: Submit keys to Server

File: Corona-Warn-App/src/main/res/values/strings.xml [hunk 7]
```diff
@@ -368,80 +368,82 @@
...
+    <string name="submission_done_contagious">Sie sind ansteckend.</string> <!-- final -->
+    <string name="submission_done_isolate">Bitte isolieren Sie sich von anderen Personen.</string> <!-- final -->
+    <string name="submission_done_button_done">Fertig</string> <!-- final -->
 
     <!-- ####################################
            Just for Development
     ###################################### -->
     <string name="lorem_ipsum">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</string>
-    <!-- TODO test fragment only - remove after test fragment is deleted-->
+    <!-- TODO test fragment only - remove after test fragment s deleted-->
```
Comment (m1ga) [inline Corona-Warn-App/src/main/res/values/strings.xml:446]: remove after test fragment is deleted

**Comment by m1ga** [comment]
I know some texts are not final yet but even in the final strings are some mistakes. Left some comments, hope it helps

**Review by harambasicluka — DISMISSED** [review]

**Review by harambasicluka — APPROVED** [review]

**Pull request merged**